src/System.Net.HttpListener/ref/System.Net.HttpListener.cs

```diff
@@ -128,6 +139,16 @@ public sealed partial class HttpListenerResponse : System.IDisposable
         public void SetCookie(System.Net.Cookie cookie) { }
         void System.IDisposable.Dispose() { }
     }
+    public partial class HttpListenerTimeoutManager
+    {
+        private HttpListenerTimeoutManager() { }
```

Comment:
I think genapis itself prefers to use "internal" when it generates these special parameterless constructors. I guess it doesn't matter but in this case it actually is internal in the impl.